vue/src/vueTextMask.js

```diff
@@ -1,24 +1,42 @@
-import createTextMaskInputElement from '../../core/src/createTextMaskInputElement'
+import createTextMaskInputElement from '../../core/src/createTextMaskInputElement.js'
 
 export default {
-  textMaskInputElement: null,
 
-  inputHandler({target: {value}}) {
-    return this.textMaskInputElement.update(value)
-  },
+  // Input event callback will be stored in this variable
+  // because we'll need it for unbind method
+  callback: null,
```

Comment:
Looking at how you're storing the function at `binding.def.callback` below, it doesn't look like you really need this line?